### DISCUSSION TITLE
Leafmap set catalogues for STAC

### DISCUSSION THREAD

**Discussion by jessetanimu** [giswqs/geog-312#158] [Q&A] [answered]
This is with respect to the GEOG-312 class
I am currently having errors in the in the last section of leafmap.  i did not change anything in the original code but I am getting the below error message
Retrieving collections...
Client.open() takes from 2 to 4 positional arguments but 8 were given
cannot unpack non-iterable NoneType object
No collections found.

**Comment by giswqs** [comment, marked as answer]
Try updating leafmap to the latest version.
